### PR TITLE
Feature bp 741 replace movai classes with flow initiator

### PR DESCRIPTION
**Pull request by erezz-mov-ai**
jira:
https://movai.atlassian.net/browse/BP-741?atlOrigin=eyJpIjoiZDVkNzNlMDZkMTIwNDcwNGFmMDAwMmIwNTk5MWQwMDEiLCJwIjoiaiJ9
- [x] Make sure you are opening from a **topic/feature/bugfix branch**
- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

File: flow_initiator/__main__.py [hunk 1]
```diff
@@ -11,6 +11,6 @@
     use `python3 -m spawner` to launch
 """
 
-from . import main
+from flow_initiator import main
```
Comment (Mograbi) [inline flow_initiator/__main__.py:14]: this need to be tested

File: flow_initiator/spawner/async_movaicore.py [hunk 2]
```diff
@@ -17,14 +17,18 @@
...
+
+from movai_core_shared.logger import Log, LogAdapter
+
+from dal.scopes.robot import Robot
+from dal.models.lock import Lock
 from dal.movaidb import RedisClient
-from dal.scopes import Robot
-from .async_spawner import Spawner
+
+from flow_initiator.spawner.async_spawner import Spawner
```
Comment (Mograbi) [inline flow_initiator/spawner/async_movaicore.py:27]: when it was relative it does not import all of the inits in the whole path
so I think it's better to preserve the old relative way of import

**Review by Mograbi — CHANGES_REQUESTED** [review]

File: flow_initiator/spawner/flowmonitor.py [hunk 3]
```diff
@@ -125,8 +125,8 @@ def transition(
                 nodes_to_transit, self.active_flow, transition_msg=transition_msg
             )
 
-        except Exception as e:
-            LOGGER.error(e)
+        except Exception:
+            LOGGER.error()
             return []
```
Comment (erezz-mov-ai) [inline flow_initiator/spawner/flowmonitor.py:130]: restore print exception

File: flow_initiator/spawner/async_movaicore.py (hunk 2 above)
Comment (erezz-mov-ai) [reply flow_initiator/spawner/async_movaicore.py:27]: done

**Review by Mograbi — APPROVED** [review]

**Review by Mograbi — APPROVED** [review]

**Pull request merged**